plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -177,28 +176,15 @@ class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
     handler.removeCallbacks(commitChangesRunnable)
   }
 
-  @VisibleForTesting(otherwise = PRIVATE)
-  internal fun performMapJump(cameraOptions: CameraOptions) {
+  private fun performMapJump(cameraOptions: CameraOptions) {
     if (lastCameraOptions == cameraOptions) {
       return
     }
     // move native map to new position
-    try {
-      mapCameraManagerDelegate.setCamera(cameraOptions)
-      // notify listeners with actual values
-      notifyListeners(mapCameraManagerDelegate.cameraState)
-      lastCameraOptions = cameraOptions
-    } catch (e: Exception) {
-      logE(
-        TAG,
-        "Exception while setting camera options : ${e.message} CameraOptions = $cameraOptions"
-      )
-    } catch (error: Error) {
```

Comment:
we were catching both `Exception` and `Error`, would it behave differently since gl-native only catches `Error` I guess?

Reply:
that's correct, we are also catching `Exceptions`. while native only cath `Error`. 
should we just keep exceptions here and let native handle `Error`? cc @mapbox/maps-android

Reply:
imo we should not catch anything - if user does pass lat as NaN - it's pretty logical to actually fail with exception thrown from gl-native.
even if we decide to catch it - then it's logical to move catching it in `MapboxMap` and not only in `CameraAnimationsPluginImpl.kt`.
I would lean to removing catching exceptions from our end - e.g. in gestures code we already check points for NaN values and that's the correct path